assets/stylesheets/kitten/components/footer/_footer-lendo.scss

```diff
@@ -0,0 +1,75 @@
+@mixin k-FooterLendo {
+  .k-FooterLendo__network__text {
+    text-align: center;
+
+    @include k-media-min('l') {
+      text-align: left;
+    }
+  }
+
+  .k-FooterLendo__network__followText {
+    text-align: center;
+
+    @include k-media-min('l') {
+      text-align: right;
+      margin-right: k-px-to-rem(15px);
+    }
+  }
+
+  .k-FooterLendo__network__follow {
+    align-self: center;
+  }
+
+  .k-FooterLendo__network__logo {
+    @include k-media-min('l') {
+      display: flex;
+      justify-content: flex-end;
+    }
+  }
+
+  .k-FooterLendo__notice__copyright {
+    text-align: center;
+
+    @include k-media-min('l') {
+      text-align: right;
+      align-self: flex-end;
+      margin-top: 0;
+    }
+  }
+
+  .k-FooterLendo__notice__logo {
+    @include k-media-min('l') {
+      display: flex;
+      text-align: left;
+    }
+  }
+
+  .k-FooterLendo__logo__img {
+    text-align: center;
+
+    > img,
+    > svg {
+      vertical-align: middle;
+    }
+
+    @include k-media-min('l') {
+      > img,
+      > svg {
+        display: block;
+        vertical-align: top;
+      }
+
+      line-height: 4rem;
+      margin-bottom: 0;
+      margin-right: 1rem;
```

Comment:
Est-ce que c'est pas plus clair de mettre `k-px-to-rem(16px)` à la place de `1rem` ?